lms/djangoapps/certificates/tests/test_signals.py

```diff
@@ -222,6 +222,63 @@ def test_cert_already_generated(self):
                 mock_generate_certificate_apply_async.assert_not_called()
 
 
+@ddt.ddt
+class FailingGradeCertsTest(ModuleStoreTestCase):
+    """
+    Tests for marking certificate notpassing when grade goes from passing to failing
```

Comment:
`Also testing to make sure if the certs are in statuses other than passing, the signal have no effect on the cert status`